src/Storages/ColumnsDescription.cpp

```diff
@@ -383,6 +383,15 @@ NamesAndTypesList ColumnsDescription::getEphemeral() const
     return ret;
 }
 
+NamesAndTypesList ColumnsDescription::getWithDefaultExpression() const
```

Comment:
Looking at the [docs](https://clickhouse.com/docs/en/sql-reference/statements/create/table/#default-values), a column with default expression can be of type "default", "materialized", "ephemeral" or "alias". 

The function name "getWithDefaultExpression()" is a bit misleading ... it restricts the results to only "default"-type columns.

Reply:
So, we've found out that KafkaStorage doesn't support MATERIALIZED type too, so I removed predicate for "default" type column and from now a function`getWithDefaultExpression` shouldn't be misleading